### PR TITLE
Mass micro optimization, code cleaning, scientific accuracy and cryogenic fuels

### PR DESCRIPTION
**Pull request by TheFakeElon**
## About The Pull Request

I probably should have atomized (haha) cryogenic fuels but eh.

## 1)
The most notable change is replacing ""jet"" fuel with cryogenic fuel. Fighter fuel must now be made at a low temperature and _kept_ at that low temperature to avoid unscheduled disassembly of objects and people near the container.

**Wiki changes:**

**Cryogenic Tyrosene**
[chem recipe is the same]
required temp = 40k or below

**Tyrosense Explosion**
required temp = 130k+
slightly more powerful than a potassium explosion

## 2)
Fixes a bunch of capitalized names that weren't disclaimed as `\improper`.

## 3)
Removes (almost) all references to fighters being "air" or "aviation" vehicles

## 4)
General lore accuracy improvements such as renaming sonar to radar, reactors now using uranium 235 instead of 238, etc.

## 5)
General code readability and efficiency tweaks 

## Why It's Good For The Game

cryogenic fuel is more accurate AND fun
The rest of the changes are code/LORE improvements

## Changelog
:cl:
refactor: Aviation Fuel has been replaced with Cryogenic Fuel, try to keep it cold
fix: Fixes a heap of grammar bugs for ships that start with a capital letter
code: General code readability and efficiency improves
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
image
image

File: nsv13/code/modules/power/stormdrive.dm [hunk 1]
```diff
@@ -1598,7 +1599,7 @@ Control Rods
 /////// ANOMALIES ///////
 
 /obj/effect/anomaly/stormdrive //PARENT
-	name = "Stormdrive Anomaly - PARENT "
+	name = "\improper Stormdrive Anomaly - PARENT "
```
Comment (Karmic-Skink) [inline nsv13/code/modules/power/stormdrive.dm:1602]: Really.

**Comment by Karmic-Skink** [comment]
Okay so you have changes to maps that have current PRs open already, expect them to take priority.

File: nsv13/code/modules/power/stormdrive.dm (hunk 1 above)
Comment (TheFakeElon) [reply nsv13/code/modules/power/stormdrive.dm:1602]: I see caps I improper

**Comment by TheFakeElon** [comment]
@Karmic-Skink This just changes paths

File: nsv13/code/modules/power/stormdrive.dm (hunk 1 above)
Comment (Karmic-Skink) [reply nsv13/code/modules/power/stormdrive.dm:1602]: I see a parent item that never spawns.

**Comment by covertcorvid** [comment]
Recompiling TGUI will fix your linter problem, not sure why you've got null gas mixtures in the test run though

**Comment by TheFakeElon** [comment]
how are there null atmos runtimes

**Comment by TheFakeElon** [comment]
This will also need wiki changes :smirk:

**Review by Kmc2000 — APPROVED** [review]

**Pull request merged**